src/crimson/os/seastore/seastore.h

```diff
@@ -433,6 +439,17 @@ class SeaStore final : public FuturizedStore {
       lat.sample_sum += std::chrono::duration_cast<std::chrono::milliseconds>(dur).count();
      }
 
+    uint32_t get_max_object_size(const Onode &onode) const {
```

Comment:
Is there a concept of per-onode max-object-size, or is it the current size of an onode, or the 
"expected" size of an onode?

IIUC the max-object-size is a global value set from configuration, .

Reply:
There's a max object size in the configuration, but there's a librados hint that allows the client application to notify the OSD (and therefore seastore) of the expected size of a specific object.  I'd guess the intent here is for get_max_object_size() to reflect the hinted size, if any.  I'd suggest modifying it to get_hinted_object_size(onode) with a special value for no hint.